### DISCUSSION TITLE
Java is not opening jar files.

### DISCUSSION THREAD

**Discussion by edgarjsanchez** [microsoft/openjdk#296] [Q&A] [answered]
Hi:

Windows 11 ARM (running in parallels on Mac M1 Max).

Installed MS Java 17 from MS:

C:\tmp>java -version
openjdk version "17.0.3" 2022-04-19 LTS
OpenJDK Runtime Environment Microsoft-32931 (build 17.0.3+7-LTS)
OpenJDK 64-Bit Server VM Microsoft-32931 (build 17.0.3+7-LTS, mixed mode)

C:\tmp>echo %JAVA_HOME%
C:\Program Files (Arm)\Microsoft\jdk-17.0.3.7-hotspot\

My problem is trying to open a Jar file. Nothing happens. No error but the task manager show a couple of Java processes that immediately close.

I can run simple java programs:
c:\tmp>java sample3.java
Hello World!

But trying to open a jar produces no results.

C:\tmp>java -jar BBjBaristaAddonDemos2202_07-06-2022_1802.jar

Any help will be greatly appreciated.

ES

**Comment by karianna** [comment, marked as answer]
I've created #298 to track this.

**Comment by dsgrieve** [comment]
`-verbose:class` can sometimes help in situations like this.
